include/pops/power_law_kernel.hpp

```diff
@@ -64,8 +69,14 @@ class PowerLawKernel
      */
     double pdf(double x)
     {
-        if (x <= 0 || xmin == 0 || alpha <= 1.0) {
-            return 0;
+        if (x < 0) {
+            throw std::invalid_argument("x cannot be less than 0.0");
+        }
+        // x = distance to center and is always 0 for the center cell
+        // so have to add xmin to account for minimum x values
+        x = x + xmin;
```

Comment:
I am not sure about this addition. I think xmin functions similarly to the locator parameter so ideally we could set this to zero like all the of other locator parameters, but the function is undefined when xmin is zero. Also x needs to be greater than or equal to xmin, but x will always be zero for the center cell so this causes an error because x is less than xmin. Do you think it is better to leave in this line (i.e shift the starting distance) or set xmin to some value close to zero (i.e 0.001)?

Reply:
I think that this is the best solution. I realized that even if xmin is hard-coded, x still must be greater than xmin or the result is invalid. Also, I tested different values of xmin (i.e 0.001 vs 0.0001) and it strongly effects the distribution between x-values 0-2 by ~0.10.

Reply:
I agree that this is the best way forward.